### PR TITLE
Updated Docs Links to 2.0 structure

### PR DESCRIPTION
**Pull request by b-quachtran**
**Proposed changes**:
- Updated docs links in source to new 2.0 structure
- close https://github.com/RasaHQ/rasa/issues/6556

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by b-quachtran** [comment]
Hmm, CI checks are being weird because of the updated links in the test config files. I think it's confusing the hashtag in the URL as a comment in `https://rasa.com/docs/rasa/tuning-your-model#how-to-choose-a-pipeline`. I'm going to take a look at the test script (`tests/importers/test_autoconfig.py`) and figure out what's going on.

File: rasa/constants.py [hunk 1]
```diff
@@ -29,16 +29,16 @@
...
+DOCS_URL_EVENT_BROKERS = DOCS_BASE_URL + "/event-brokers"
 DOCS_URL_PIKA_EVENT_BROKER = DOCS_URL_EVENT_BROKERS + "#pika-event-broker"
-DOCS_URL_TRACKER_STORES = DOCS_BASE_URL + "/api/tracker-stores/"
-DOCS_URL_PIPELINE = DOCS_BASE_URL + "/nlu/choosing-a-pipeline/"
-DOCS_URL_COMPONENTS = DOCS_BASE_URL + "/nlu/components/"
-DOCS_URL_MIGRATION_GUIDE = DOCS_BASE_URL + "/migration-guide/"
+DOCS_URL_TRACKER_STORES = DOCS_BASE_URL + "/tracker-stores"
+DOCS_URL_PIPELINE = DOCS_BASE_URL + "/tuning-your-model"
+DOCS_URL_COMPONENTS = DOCS_BASE_URL + "/components"
+DOCS_URL_MIGRATION_GUIDE = DOCS_BASE_URL + "/migrate-from/"
```
Comment (erohmensing) [inline rasa/constants.py:41]: This is different -- migraation guide is migrating from one version to another, migrate from is migrating from other tools to rasa

File: rasa/cli/x.py [hunk 2]
```diff
@@ -289,8 +289,7 @@ def _validate_rasa_x_start(args: argparse.Namespace, project_path: Text):
         cli_utils.print_error_and_exit(
             "This directory is not a valid Rasa project. Use 'rasa init' "
             "to create a new Rasa project or switch to a valid Rasa project "
-            "directory (see http://rasa.com/docs/rasa/user-guide/"
-            "rasa-tutorial/#create-a-new-project)."
+            "directory (see http://rasa.com/docs/rasa/installation#quick-installation)."
```
Comment (erohmensing) [inline rasa/cli/x.py:292]:
```suggestion
            "directory (see https://rasa.com/docs/rasa/command-line-interface#rasa-init)."
```

File: data/configs_for_docs/example_for_suggested_config_after_train.yml [hunk 3]
```diff
@@ -3,7 +3,7 @@ language: en
 pipeline:
 # # No configuration for the NLU pipeline was provided. The following default pipeline was used to train your model.
 # # If you'd like to customize it, uncomment and adjust the pipeline.
-# # See https://rasa.com/docs/rasa/nlu/choosing-a-pipeline/ for more information.
+# # See https://rasa.com/docs/rasa/tuning-your-model#how-to-choose-a-pipeline for more information.
```
Comment (erohmensing) [inline data/configs_for_docs/example_for_suggested_config_after_train.yml:6]:
```suggestion
# # See https://rasa.com/docs/rasa/next/model-configuration#suggested-config for more information.
```

This should go for anything that is suggested by autoconfig

**Review by erohmensing — CHANGES_REQUESTED** [review]
Looking good so far! left some comments on yours, going in and catching some you missed now :)

**Comment by b-quachtran** [comment]
@erohmensing I didn't include `next/` in the URL's since I assumed that after the docs migration it won't be needed. Should it be left in for now (e.g. https://rasa.com/docs/rasa/next/custom-actions)?

**Comment by erohmensing** [comment]
No, if i merged that in its definitely a mistake 🙈

**Review by erohmensing — APPROVED** [review]
Amazing! Thank you for helping out!

**Pull request merged**